luma/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Richard Hull and contributors
+# See LICENSE.rst for details.
+
+from pkgutil import extend_path
```

Comment:
This is to allow the namespaced packages to work properly on python 2.7. As far as I can tell it is not necessary on 3.4+, but also doesnt do any harm